code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm

```diff
@@ -96,6 +96,30 @@
 
 		occupant.visible_message("<span class='danger'>[occupant] crashed into \the [A]!</span>")
 
+/obj/structure/chair/wheelchair/plasteel
+	name = "Hardened Wheelchair"
+	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than a typical wheelchair and they are very resistant to acids and corrosion, thanks to that they can be used in various environments without the need to worry about safety."
```

Comment:
```suggestion
	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than a typical wheelchair and is very resistant to acid and corrosion. Thanks to this it can be used in various dangerous environments without the need to worry about safety."
```